### ISSUE TITLE
Feature request: Vault support for item currency

### ISSUE THREAD

**Issue by KickSquareMC** [SirLeezus/OneStopShop#11]
I love that you added an item currency to onestopshop! Would it be possible to make the currency a full vault-compatible currency, so that you can use the items for any other plugins that use economy? For example, if the item currency uses emeralds, that I could use those emeralds to buy something from another plugin, similar to a plugin called "TheNewEconomy". Thanks, if you have any questions what I mean reply here or add me on discord, `KickSquare#7689`

**Comment by LeeTheTech** [comment]
Yeah, it would just be a matter of adding meta data checking vs just the material, not sure when I'll find time but I'll add it to my list.

**Comment by KickSquareMC** [comment]
Great thanks, feel free to reply to this thread when it's implemented!

**Comment by GodsBrocc** [comment]
Do you mean being able to use both emeralds and vault in unison?

I've been trying to figure out how to use a material currency and vault, meaning - an item in a category can cost $ and another item can cost emeralds. Let me know if this is possible to do, thank you for the great plugin!
